kumascript/src/api/web.js

```diff
@@ -75,8 +107,10 @@ module.exports = {
               current: subpath,
             }
           );
+          flawAttribute = ` data-flaw-src="${util.htmlEscape(
+            flaw.macroSource
+          )}"`;
         }
-        flawAttribute = ` data-flaw-src="${util.htmlEscape(flaw.macroSource)}"`;
```

Comment:
Hm, by moving `flawAttribute`, it will provide a `data-flaw-src` for the `wrong-xref-macro` flaw, but not for the `redirected-link` flaw. Don't we need the `data-flaw-src` for the `redirected-link` flaw as well? I added a suggestion for the `redirected-link` flaw code path to cover that case.